data/filedefs/filetypes.cobol

```diff
@@ -16,7 +16,8 @@ quotedidentifier=identifier_2
+primary=accept add advancing after alphabet also alternate and are ascending assign at author before bottom call cancel cbl cd cf ch class clock-units close cobol code code-set collating communications compute configuration continue control converting corr corresponding count currency data de declare debug-contents debug-item debug-line debug-name debug-sub-1 debug-sub-2 debug-sub-3 debugging delaratives delete delimited delimiter depending descending destination detail disable display divide division down duplicates egi eject else emi enable end end-add end-call end-compute end-delete end-divide end-evaluate end-exec end-if end-multiply end-of-page end-perform end-read end-receive end-return end-rewrite end-search end-start end-string end-subtract end-unstring end-write environment error esi evaluate every exception exec exit extend fd fetch file-control final first footing for from generate giving go goback group heading i-o i-o-control identification if indicate initial initialize initiate input-output insert inspect installation into is just justified last leading left linkage lock memory merge message move multiple multiply native negative next no not object-computer off on open optimize optional or order overflow padding perform pf ph pic picture plus position positive printing procedure proceed program program-id purge recording rd read receive redefines reel relative release remainder removal replace replacing report reporting reports rerun reserve reset return returning reversed rewind rewrite rf rh right rounded same sd search section security segment segment-limited select send sentence separate sequence sequential set sign skip sort sort-merge source source-computer special-names sql start string subtract sum suppress symbolic sync synchronized tallying tape terminal terminate test text then through thru to top trailing type union unit unstring until update upon usage use using varying when where with working-storage write
```

Comment:
Isn't `delaratives` a typo?

Reply:
My most sincere apologies, I just rechecked the file and it's my only mistake, I'll fix it : it's obviously DECLARATIVES

Reply:
PS : It seems that the actual filedef.cobol has the same error

Reply:
yes, we currently have the issue already